Alert.css

```diff
@@ -0,0 +1,77 @@
+:root {
+  --color: black;
+}
```

Comment:
Don't just specify the color name here. Not a good variable name.

Reply:
Noted and Updated with New color Name